### PR TITLE
Add bundlesize and monitor precompiled assets and packs - take 2

### PR DESCRIPTION
**Pull request by rhymes**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This is the same as https://github.com/thepracticaldev/dev.to/pull/6076 but as a regular PR, apparently we weren't able to trigger Travis for unknown reasons

Please read the other's description

Closes https://github.com/thepracticaldev/dev.to/issues/5577

### CODE REVIEW

File: bundlesize.config.json [hunk 1]
```diff
@@ -21,107 +21,107 @@
       "maxSize": "23 kB"
     },
     {
-      "path": "public/packs/analyticsArticle-*.js",
+      "path": "public/packs*/analyticsArticle-*.js",
```
Comment (maestromac) [inline bundlesize.config.json]: Ah, so that's why it was failing. I tested it locally and seems to be working well!

**Comment by rhymes** [comment]
@citizen428 not really in reality, the build failed because the config file has not been updated and I don't see the [bundlesize bot](https://github.com/siddharthkp/bundlesize#build-status-for-github) yet 👀 

I'm updating the conf file now

**Comment by citizen428** [comment]
@rhymes Sorry, I guess I was still too sleepy. Please re-request a review when you're done with your changes.

**Comment by rhymes** [comment]
@maestromac @citizen428 good news I think.

The build failed because one of the compile files is bigger than the limit I set (well, it passed locally so we might need to give a little more leeway to some of them)

It also told us what is the likely reason we're not seeing the bot yet:

```text
$ /home/travis/build/thepracticaldev/dev.to/node_modules/.bin/bundlesize

 WARN  github token not found

    You are missing out on some cool features.

    Read more here: https://github.com/siddharthkp/bundlesize#2-build-status
```

I think Nick should have a look at this before we decide to merge it, also because we should discuss expectations and performance budgets with the team as this is going to fail builds

**Comment by maestromac** [comment]
@rhymes I should've realized this sooner. For this check to work on fork repositories we need to make the keys public (ie. right inside `travis.yml` and unencrypted). I don't think there's any security implication here that we need to be concerned of. Would you agree?

**Comment by rhymes** [comment]
@maestromac the key seems to be limited to the `repo:status` scope, but I'd ask the [bundlesize](https://github.com/siddharthkp/bundlesize) team to make sure it's not a problem!

**Comment by citizen428** [comment]
@rhymes What's the status on this? Should we close it out for now or will you be working on this again?

**Comment by rhymes** [comment]
@citizen428 I think we were trying to figure out how to deal with the key, see this issue opened by https://github.com/siddharthkp/bundlesize/issues/356

Just asked about updates.

**Comment by rhymes** [comment]
@maestromac what do you think of https://github.com/siddharthkp/bundlesize/issues/356#issuecomment-628511235 ?

**Comment by maestromac** [comment]
@rhymes I'm going to give their https://github.com/siddharthkp/bundlesize2 a try

**Comment by mstruve** [comment]
Closing since we have not had any activity on this PR for a couple of months. @rhymes feel free to update the code and revisit this idea if you think it's a must have!

**Pull request closed without merging**